common/etc/nginx/templates/default.conf.template

```diff
@@ -23,6 +24,7 @@ js_var $indexIsEmpty true;
 js_set $s3auth s3gateway.s3auth;
 js_set $awsSessionToken awscredentials.sessionToken;
 js_set $s3uri s3gateway.s3uri;
+js_var $defaultHostName 'nginx-s3-gateway';
```

Comment:
What's the purpose of setting this variable?

Reply:
This is the purpose of option 2 in this link: https://github.com/nginxinc/nginx-s3-gateway/pull/126#discussion_r1190622214

Reply:
I have removed this variable, and you could find my answer in the https://github.com/nginxinc/nginx-s3-gateway/pull/126#discussion_r1193013453.